pkg/module_manager/models/modules/values_storage.go

```diff
@@ -242,6 +240,19 @@ func (vs *ValuesStorage) GenerateNewConfigValues(configV utils.Values, validate
 		if err != nil {
 			return nil, err
 		}
+
+		mergedWithOpenapiDefault := mergeLayers(
+			utils.Values{},
+			vs.staticConfigValues,
+			vs.openapiDefaultsTransformer(validation.ConfigValuesSchema),
+			configV,
+		)
+		mergedWithOpenapiDefault = mergedWithOpenapiDefault.Copy()
```

Comment:
I don't think that we have to copy this, because we don't return this variable, only use it for validation

Reply:
also I don't think that we need to validate `merged` values without openapi defaults. It seems like double job. Please remove this
```
err := vs.validateConfigValues(merged)
```

Reply:
done